ecl2df/common.py

```diff
@@ -114,13 +115,140 @@
 logger: logging.Logger = logging.getLogger(__name__)
 
 
-def write_dframe_stdout_file(
+def set_name_from_args(args):
```

Comment:
I don't quite get what this is doing from reading the function name in the context of common.py. Can it get a more descriptive name, optionally an example in the docstring?